### ISSUE TITLE
Legacy add-on cannot be updated once a WE submissions has been submitted (edge case)

### ISSUE THREAD

**Issue by spoji** [mozilla/addons#373]
### Describe the problem and steps to reproduce it:
https://addons.mozilla.org/en-US/editors/review/dictcc-translation wanted a quick review similar to legacy version 4.2.0. Unfortunately, as there was already two submitted versions of his add-on migrated to WE, the dev could do upload a new legacy build.

### What happened?
The dev stumble on this error : 
"You cannot update a WebExtensions add-on with a legacy add-on. Your users would not be able to use your new version because Firefox does not support this type of update".

It's misleading as no WE builds were approved and pushed to public.

### What did you expect to happen?
If no WE updates are public, the system should accept legacy uploads.

### Anything else we should know?
There is an workaround right now for this that admins are able to do, TheOne can fill you in on this if needed.

I understand this bug will probably go wontfix, I'm filling up an issue we had in #addon-reviewers with a dev. We have an alternative for this and this issue will be solved by itself once Firefox 57 lands.

**Comment by John-Nagle** [comment]
You can work around this by uploading your WebExtensions add-on under a new name, in experimental mode, for test and initial review. Once it's working and reviewed, it can be uploaded again under the old name. Of course, this doubles AMO's workload, but it works.

**Comment by jvillalobos** [comment]
Legacy extensions are no longer supported.
